src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -196,7 +196,7 @@ class Request
     /**
      * @var string
      */
-    protected $format;
+    protected $requestedResponseFormat;
```

Comment:
Renaming a protected property is a BC break. This is not allowed. See https://symfony.com/doc/current/contributing/code/bc.html for details about which kind of changes are allowed or no to respect the BC policy.

Reply:
Will revert that.